pkgdb2client/admin.py

```diff
@@ -378,7 +389,14 @@ def do_process(args):
     '''
     LOG.info("user   : {0}".format(args.username))
 
-    for actionid in args.actionid:
+    args.status = "Awaiting Review"
+    if not args.actionid:
+        print 'Processing all requests with status: %s' % args.status
```

Comment:
On Mo, Nov 16, 2015 at 10:38:53 -0800, Pierre-Yves Chibon wrote:
> > @@ -378,7 +389,14 @@ def do_process(args):
> >      '''
> >      LOG.info("user   : {0}".format(args.username))
> >  
> > -    for actionid in args.actionid:
> > +    args.status = "Awaiting Review"
> > +    if not args.actionid:
> > +        print 'Processing all requests with status: %s' % args.status
> 
> Should we ask for a yes/no confirmation here?

Since there will be a confirmation for each request, I do not think an
extra confirmation is needed here.